docs/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"

```

Comment:
Instead of committing this file, I think that `docs/.bundle` should be in the `.gitignore`. See https://github.com/Orange-OpenSource/ods-jekyll-theme/blob/main/.gitignore.